src/llm/http_llm_calculator.cc

```diff
@@ -162,6 +168,22 @@ class OpenAIChatCompletionsRequest {
             this->stream = it->value.GetBool();
         }
 
+        it = this->doc.FindMember("stream_options");
+        if (it != this->doc.MemberEnd()) {
+            if (!this->stream)
+                return absl::InvalidArgumentError("stream_options provided, but stream not set to true");
+            if (!it->value.IsObject())
+                return absl::InvalidArgumentError("stream_options is not an object");
+            auto streamOptionsObj = it->value.GetObject();
+
+            it = streamOptionsObj.FindMember("include_usage");
```

Comment:
Should we validate that there are unexpected keys in the map?

Reply:
Yeah it would be good to log options we do not support.